packages/riverpod_cli/fixtures/unified_syntax/golden/widgets.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:riverpod/riverpod.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class Counter extends StateNotifier<int> {
+  Counter() : super(1);
+  void increment() => state++;
+  void decrement() => state--;
+}
+
+final counterProvider = StateNotifierProvider<Counter, int>((ref) => Counter());
+
+class ConsumerWatch extends ConsumerWidget {
+  const ConsumerWatch({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context, WidgetReference ref) {
+    // ignore: unused_local_variable
+    final countNotifier = ref.watch(counterProvider.notifier);
+    final count = ref.watch(counterProvider);
+    return Center(
+      child: Text('$count'),
+    );
+  }
+}
+
+class StatelessRead extends ConsumerWidget {
+  const StatelessRead({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context, WidgetReference ref) {
+    return Center(
+      child: ElevatedButton(
+        onPressed: () {
+          ref.read(counterProvider);
+          ref.refresh(counterProvider);
+        },
+        child: const Text('Counter'),
+      ),
+    );
+  }
+}
+
+class StatefulConsumer extends StatefulWidget {
+  const StatefulConsumer({Key? key}) : super(key: key);
+
+  @override
+  _StatefulConsumerState createState() => _StatefulConsumerState();
+}
+
+class _StatefulConsumerState extends State<StatefulConsumer> {
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Consumer(
+        builder: (context, ref, child) {
+          return Text('${ref.watch(counterProvider)}');
+        },
+      ),
+    );
+  }
+}
+
+class HooksWatch extends ConsumerHookWidget {
```

Comment:
Could you test custom hooks?

```dart
T useMyHook() {
  <...>
 useProvider(myProvider);
}

void useAnotherHook() {
  useMyHook();
}

Widget build(context) {
  useAnotherHook();
}
```

should become:

```dart
T useMyHook(WidgetReference ref) {
  <...>
  ref.watch(myProvider);
}

void useAnotherHook(WidgetReference ref) {
  useMyHook(ref);
}

Widget build(context. WidgetReference ref) {
  useAnotherHook(ref);
}
```

Reply:
Yeah, I will do this, it will take some more work but it should be doable